go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/artefactual-sdps/enduro
 
-go 1.23.6
+go 1.24.2
 
 require (
-	ariga.io/atlas v0.31.0
+	ariga.io/atlas v0.32.0
```

Comment:
Did you have to manually sync this version of the atlas module with the atlas binary used for code generation, or does bine do some magic to sync the versions?

Reply:
Manually :cry:

Reply:
I made a note to remind me of this, it sounds doable.